src/DonorProfiles/resources/js/app/components/badge/style.scss

```diff
@@ -0,0 +1,19 @@
+.give-donor-profile-badge {
+	background: #fff;
+	box-shadow: 0 1.60377px 6.41509px rgba(0, 0, 0, 0.11);
```

Comment:
Assuming this was pulled straight out of Figma, but I don't think that CSS supports _partial_ pixels. My [research](https://stackoverflow.com/a/4309051) suggests that when using `px` the value is either truncated or rounded.

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/5444/commits/b41203de3d176c9cea0c1c5650854cb926246197